src/lib/services/identity.ts

```diff
@@ -1,8 +1,59 @@
 import { NotFound, HttpResponse } from '../error-handler'
 import env from '../../env'
+export const serviceState = {
+  UP: 'up',
+  DOWN: 'down',
+  ERROR: 'error',
+} as const
+type serviceStateType = typeof serviceState
+export type SERVICE_STATE = serviceStateType['UP' | 'DOWN' | 'ERROR']
+export type Status = {
+  status: SERVICE_STATE
+  detail: Record<string, unknown> | null
+}
 
 const URL_PREFIX = `http://${env.IDENTITY_SERVICE_HOST}:${env.IDENTITY_SERVICE_PORT}/v1`
 
+export default class IdentityClass {
+  constructor() {}
+
+  getStatus = async (): Promise<Status> => {
+    try {
+      const res = await getMemberBySelf()
```

Comment:
There should be a `/health` endpoint on the identity service that surfaces information about the identity instance. That should be better than using the self endpoint